Jenkinsfile.aws-test

```diff
@@ -61,6 +61,9 @@ node(NODE) {
                     aws ec2 modify-image-attribute \
                         --image-id ${ami_intermediate} \
                         --launch-permission '{"Add":[{"UserId":"${AWS_CI_ACCOUNT}"}]}'
+                    aws ec2 modify-image-attribute \
+                        --image-id ${ami_intermediate} \
+                        --launch-permission "Add=[{Group=all}]
```

Comment:
We can drop the previous invocation of `modify-image-attribute` since now anyone can access right?

Reply:
(Or alternatively think of this as just changing the argument to `--launch-permission)

Reply:
Quoting looks a bit off -- I'm guessing it should be `'{"Add":[{"Group":"all"}]}'` ?

Reply:
I agree. I think @jlebon has the right idea on the quoting.

```
--launch-permission "Add=[{Group=all}]
```

becomes

```
--launch-permission '{"Add":[{"Group":"all"}]}'
```

Reply:
> We can drop the previous invocation of modify-image-attribute since now anyone can access right?

Yup! We shouldn't need to do that any longer. @kikisdeliveryservice If you don't mind, add another commit in this PR to remove that section.

Reply:
Thanks @ashcrow @jlebon @cgwalters ! I'll make those changes and repush.